jplag.frontend.text/src/main/java/de/jplag/text/TokenPosition.java

```diff
@@ -0,0 +1,8 @@
+package de.jplag.text;
+
+public record TokenPosition(
+    int line,
+    int column,
+    int length
+) {
+}
```

Comment:
Please provide some minimal documentation here

Reply:
Or actually, you could also just remove this record, adjust the constructor of `TextToken` to take these 3 values and pass them directly. As far as I see, the record has no significance.